spec/factories/books.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :book do
+    
```

Comment:
If you create a factory, then it should create a valid Book instance. Try creating one using Faker gem.

Reply:
Done